### PR TITLE
[fix bug] add optimizer.zero_grad() in  kws/utils/executor.py (#72)

### PR DESCRIPTION
**Pull request by chmod740**
I noticed that in the training stage, when optimizer.step() is called, optimizer.zero_grad() is not called, so I submit this commit to fix this bug.

### CODE REVIEW

**Review by blessyyyu — COMMENTED** [review]
I think you should add `optimizer.zero_grad()` above `loss.backward()`

**Comment by robin1001** [comment]
> I think you should add `optimizer.zero_grad()` above `loss.backward()`

I think so.

**Comment by chmod740** [comment]
Thank you for the suggestion, I have made some changes.

**Review by mlxu995 — APPROVED** [review]
